src/components/PipelineRun/PipelineRunListView/PipelineRunListRow.tsx

```diff
@@ -115,6 +115,9 @@ const BasePipelineRunListRow: React.FC<React.PropsWithChildren<BasePipelineRunLi
           '-'
         )}
       </TableData>
+      <TableData className={pipelineRunTableColumnClasses.eventType}>
+        {obj.metadata?.labels[PipelineRunLabel.COMMIT_EVENT_TYPE_LABEL] ?? '-'}
```

Comment:
This label alone is not adding enough value, Maybe we should add link to the PR also. `pull_request` is not intuitive It should `Pull request`. Link can be something like `(github/gitlab icon)org/repository#<pr-number>`.